### PR TITLE
Update fireblocks-sdk

### PR DESCRIPTION
**Pull request by vsiskin**
Updated fireblocks-sdk endpoints on nft-service ( getNFTs and getOwnedNFTs ).

Adding sort and order to both of them
vaultAccountId -> vaultAccountIds and added collectionIds (getOwnedNFTs)

### CODE REVIEW

File: fireblocks_sdk/sdk.py [hunk 1]
```diff
@@ -58,9 +58,10 @@ def get_nft(self, id: str):
 
         return self._get_request(url)
 
-    def get_nfts(self, ids: List[str], page_cursor: str = '', page_size: int = 100):
+    def get_nfts(self, ids: List[str], page_cursor: str = '', page_size: int = 100, sort: str = None, order: str = None):
```
Comment (idanya) [inline fireblocks_sdk/sdk.py:61]: what are the possible params for sort/order? is it a dynamic string? 
If that is a closed list, maybe create a ENUM for it?

Comment (vsiskin) [reply fireblocks_sdk/sdk.py:61]: ASC and DESC for order and for sort we have one possible value for now, but that will change in few days I think

We can use enum

File: fireblocks_sdk/sdk.py [hunk 2]
```diff
@@ -58,9 +59,17 @@ def get_nft(self, id: str):
 
         return self._get_request(url)
 
-    def get_nfts(self, ids: List[str], page_cursor: str = '', page_size: int = 100):
+    class GetOwnedNftsSortValues(Enum):
+        "ownershipLastUpdateTime" = "ownershipLastUpdateTime"
```
Comment (idanya) [inline fireblocks_sdk/sdk.py:63]:
```suggestion
        OWNERSHIP_LAST_UPDATE_TIME = "ownershipLastUpdateTime"
```

File: fireblocks_sdk/sdk.py [hunk 3]
```diff
@@ -106,8 +118,8 @@ def refresh_nft_ownership_by_vault(self, blockchain_descriptor: str, vault_accou
 
         return self._get_request(url, query_params=params)
 
-    def get_owned_nfts(self, blockchain_descriptor: str, vault_account_id: str, ids: List[str] = None,
-                       page_cursor: str = '', page_size: int = 100):
+    def get_owned_nfts(self, blockchain_descriptor: str, vault_account_ids: List[str] = None, ids: List[str] = None,
+                       collectionIds: List[str] = None, page_cursor: str = '', page_size: int = 100, sort: str = GetOwnedNftsSortValues[], order: OrderValues = None):
```
Comment (idanya) [inline fireblocks_sdk/sdk.py:122]:
```suggestion
                       collectionIds: List[str] = None, page_cursor: str = '', page_size: int = 100, sort: List[GetOwnedNftsSortValues] = None, order: OrderValues = None):
```

**Review by idanya — CHANGES_REQUESTED** [review]

**Review by idanya — CHANGES_REQUESTED** [review]
Please fix pipelines.

**Review by idanya — APPROVED** [review]

**Pull request merged**